### PR TITLE
[HtmlSanitizer] fix PHP config examples

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

File: html_sanitizer.rst [hunk 1]
```diff
@@ -793,7 +792,7 @@ URLs of ``<a>`` elements:
                     // specifies the allowed hosts, the attribute will be dropped if the
                     // URL contains a different host. Subdomains are allowed: e.g. the following
                     // config would also allow 'www.symfony.com', 'live.symfony.com', etc.
-                    ->allowedLinkHost('symfony.com')
+                    ->allowedLinkHosts('symfony.com')
```
Comment (OskarStark) [inline html_sanitizer.rst]: Is this really a string, I would expect and array here 🤔

Comment (javiereguiluz) [reply html_sanitizer.rst]: In the config class created by Symfony I see this:

```php
    /**
     * Allows only a given list of hosts to be used in links href attributes.
     * @default null
     * @param ParamConfigurator|mixed $value
     *
     * @return $this
     */
    public function allowedLinkHosts(mixed $value = NULL): static
    {
        $this->_usedProperties['allowedLinkHosts'] = true;
        $this->allowedLinkHosts = $value;

        return $this;
    }
```

File: html_sanitizer.rst [hunk 2]
```diff
@@ -912,7 +911,7 @@ the HTML sanitizer: ``src``, ``href``, ``lowsrc``, ``background`` and ``ping``.
 
                     // specifies the allowed hosts, the attribute will be dropped if the URL
                     // contains a different host which is not a subdomain of the allowed host
-                    ->allowedMediaHost('symfony.com') // Also allows any subdomain (i.e. www.symfony.com)
+                    ->allowedMediaHosts('symfony.com') // Also allows any subdomain (i.e. www.symfony.com)
```
Comment (mdoutreluingne) [inline html_sanitizer.rst:914]: [ `allowMediaHosts`](https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/HtmlSanitizer/HtmlSanitizerConfig.php#L219C5-L219C11) with an array as parameter

File: html_sanitizer.rst [hunk 3]
```diff
@@ -699,7 +698,7 @@ element (even if the original one didn't contain a ``rel`` attribute):
 
         $postSanitizer = new HtmlSanitizer(
             (new HtmlSanitizerConfig())
-                ->forceAttribute('a', 'rel', 'noopener noreferrer')
+                ->forceAttribute('a', ['rel' => 'noopener noreferrer'])
```
Comment (mdoutreluingne) [inline html_sanitizer.rst]: Here, the parameters are just character strings in the `HtmlSanitizerConfig` component class, [see here](https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/HtmlSanitizer/HtmlSanitizerConfig.php#L375)

Comment (xabbuh) [reply html_sanitizer.rst]: oops, good catch, reverted

File: html_sanitizer.rst (hunk 1 above)
Comment (xabbuh) [reply html_sanitizer.rst]: You are right. The CI even confirms this after merging symfony-tools/symfony-application#19.

**Review by javiereguiluz — APPROVED** [review]

**Review by mdoutreluingne — APPROVED** [review]

**Pull request merged**